pages/feed.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <THeader title="Feed">
+     <THeader :title="$t('feed.title')">
```

Comment:
where  this `feed.title` is used again?
Just only used here once time?

Reply:
I wouldn't use `$('Feed')` so it's possible to change english translation as well, to something like `News` or `Posts` any time without changing the code.

My suggestion for i18n keys is to use pattern: `<page>.<block>.<element>.<modifier>`.

Objections?

Reply:
Answer to your question: 

> where this `feed.title` is used again?

nowhere

> Just only used here once time?

yes

Reply:
For used once time, there is needed to add in locale file?

Reply:
We have 11 languages ;)

Reply:
Oh, gotcha. 👍

Reply:
I read the articles and I understand the approach for naming the keys, but I don't really understand your first sentence. Can you explain to me? @razbakov

Reply:
@Stefanacef you did everything right